include/boost/date_time/gregorian/greg_year.hpp

```diff
@@ -30,7 +30,7 @@ namespace gregorian {
   //! Generated representation for gregorian year
   typedef CV::constrained_value<greg_year_policies> greg_year_rep;
 
-  //! Represent a day of the month (range 1900 - 10000) 
+  //! Represent a year (range 1400 - 10000) 
   /*! This small class allows for simple conversion an integer value into
       a year for the gregorian calendar.  This currently only allows a
       range of 1900 to 10000.  Both ends of the range are a bit arbitrary
```

Comment:
Which is it? 1400 or 1900?   You've changed line 33 to say 1400, but line 36 says 1900.